content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,150 @@
+
+## Optimize
+
+To optimize this page, you need to know which classes are considered “critical”.
+You'll use the the [Coverage Tool](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) for that:
+
+1. In DevTools, open the [Command Menu](https://developers.google.com/web/tools/chrome-devtools/ui#command-menu), by pressing Control+Shift+P or Cmd+Shift+P (Mac).
+1. Type "Coverage" and select "Show Coverage".
+1. Click the "Reload" button, to reload the page and start capturing the coverage.
+
+<br/>
+
+<figure>
+  <img src="./coverage-unoptimized.png"
+       alt="Coverage for CSS file, showing 55.9% unused bytes." class="screenshot">
+</figure>
+
+
+Double-click on the report, to see classes marked in two colors:
+
+* Green (**critical**): These are the classes the browser needs to render the visible content (like the title, subtitle, and accordion buttons).
+* Red (**non-critical**): These styles are applied to content that's not immediately visible (like the paragraphs that appear after clicking on each button). Additionally, class definitions like "h2" are not even used on this page.
+
+With this information, you'll apply an optimization to let the browser start processing the critical styles immediately after the page loads, while deferring the load of the non-critical CSS for later:
+
+* Extract the class definitions marked with green in the report obtained from the coverage tool, and put those classes inside a `<style>` block at the head of the page:
+
+```
+<style type="text/css">
+.accordion-btn {background-color: #ADD8E6;color: #444;cursor: pointer;padding: 18px;width: 100%;border: none;text-align: left;outline: none;font-size: 15px;transition: 0.4s;}.container {padding: 0 18px;display: none;background-color: white;overflow: hidden;}h1 {word-spacing: 5px;color: blue;font-weight: bold;text-align: center;}
+</style>
+```
+
+* Then, load the rest of the classes asynchronously, by applying the following pattern:
+
+```
+<link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"> 
+<noscript><link rel="stylesheet" href="styles.css"></noscript>
+```
+
+This is not the standard way of loading CSS. Here's how it works:
+
+* `link rel="preload" as="style"` requests the stylesheet asynchronously. You can learn more about `preload` in the [Preload critical assets guide](https://web.dev/fast/preload-critical-assets).
+* The `onload` attribute in the `link` allows the CSS to be processed when it finishes loading.
+* "nulling" the `onload` handler once it is used helps some browsers avoid re-calling the handler upon switching the rel attribute.
+* The reference to the stylesheet inside of a `noscript` element works as a fallback for browsers that don’t execute JavaScript.
+
+The [resulting page](https://defer-css-optimized.glitch.me/) looks exactly like the previous version, even when most of its styles are loaded asynchronously:
```

Comment:
Small leaning to try and get into one line: "even when most styles load asynchronously". This should get all the content on one line.